samples/airport-passengers/README.md

```diff
@@ -0,0 +1,69 @@
+# Airport Passenger Numbers
+
+## Summary
+
+A Viva Connections adaptive card made for an airport to show the recent trend of daily passenger numbers for arrivals, transfers, and departures. This card is currently static and would ideally be updated to connect to real time data. It's provided here as a design template. Clicking on each of the buttons across the middle show the data for each view: an A/B/C "busy-ness" rating, number of departing passengers in thousands, number of transferring passengers, and number of arriving passengers.
+
+![passenger numbers adaptive card](https://github.com/pnp/AdaptiveCards-Templates/assets/56914706/75e2b1bd-30fa-47f6-bc50-2a2dcbe2041e)
```

Comment:
Can you add a card.png file into an assets subfolder and update this link please, like this sample - https://github.com/pnp/AdaptiveCards-Templates/tree/main/samples/yammer-liveevents/assets